lxd/node/config.go

```diff
@@ -169,16 +169,56 @@ var ConfigSchema = config.Schema{
 	"storage.images_volume":  {},
 }
 
+func validateCoreHTTPSAddress(value string) error {
+	if value == "" {
+		return nil // Deleting entry
+	}
+
+	host, _, err := net.SplitHostPort(value)
+	if err != nil {
+		return errors.Wrap(err, "Address not in form of <HOST>:<PORT>")
+	}
+
+	if shared.StringInSlice(host, []string{"::", "0.0.0.0"}) {
+		return nil
+	}
```

Comment:
Those should be valid addresses and so pass ParseIP, no?

Reply:
yes, you are right. i changed this.